.github/workflows/build.yml

```
@@ -2,18 +2,23 @@ name: Python Fire
 
 on:
   push:
-    branches:
-      - master
+    branches: ["master"]
   pull_request:
-    branches:
-      - master
+    branches: ["master"]
+
+defaults:
+  run:
+    shell: bash
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13.0-rc.2"]
+        os: ["windows-latest", "macos-latest", "ubuntu-latest"]
```

```suggestion
        os: ["macos-latest", "ubuntu-latest"]
```
Let's skip win until the build script is updated